### PR TITLE
Added a button to plot trajectories from file

### PR DESCRIPTION
**Pull request by Brudhu**
In "Trajectory" tab

(for easy visualization)

### CODE REVIEW

File: isstools/gui.py [hunk 1]
```diff
@@ -501,6 +502,22 @@ def save_trajectory(self):
             else:
                 print('\n.txt is not a valid name')
 
+    def plot_traj_file(self):
+        self.traj.load_trajectory_file('/GPFS/xf08id/trajectory/' + self.comboBox.currentText())
```
Comment (tacaswell) [inline isstools/gui.py]: Can you get to this data through the databroker?

File: isstools/gui.py [hunk 2]
```diff
@@ -501,6 +502,22 @@ def save_trajectory(self):
...
+
+        self.figure_single_trajectory.clf()
+        self.figure_single_trajectory.add_subplot(111)
+        self.canvas_single_trajectory.draw()
```
Comment (tacaswell) [inline isstools/gui.py]: `draw_idle` is usually a better function to call, it lets the GUI decide when to actually do the re-rendering.

File: isstools/gui.py (hunk 1 above)
Comment (Brudhu) [inline isstools/gui.py]: Tom, I think I can (need to write the code)... I think Eli wanted to keep parsing the files like we are doing for some reason, but I can't remember why.

File: isstools/gui.py (hunk 2 above)
Comment (Brudhu) [inline isstools/gui.py]: Great! Thanks. I will start using draw_idle.

**Pull request merged**